core/src/main/res/values/strings.xml

```diff
@@ -389,7 +389,10 @@
     <!-- Permissions -->
     <string name="alert_dialog_storage_permission_text">Please reboot your phone or restart AndroidAPS from the System Settings \notherwise Android APS will not have logging (important to track and verify that the algorithms are working correctly)!</string>
 
-    <string name="key_use_darkmode" translatable="false">use_darkmode</string>
+    <string name="key_useDarkmode" translatable="false">system</string>
```

Comment:
This has been broken, as it is now a strange `system` key, rather than `use_darkmode` which it should be.

Reply:
i commit your changes now.